contributors/guide/release-notes.md

```diff
@@ -22,14 +22,12 @@ For pull requests that require additional action from users switching to the new
     action required: your release note here
     ```
 
-For pull requests that don't need to be mentioned at release time, just write "NONE" (case insensitive):
+For pull requests that don't need to be mentioned at release time, use the `/release-note-none` Prow command to add the `release-note-none` label to the PR. You can also write the string "NONE" as a release note in your PR description:
```

Comment:
oops, looks like this shows only the first commit?

Reply:
My bad, I don't know why I read that as /release-note-action-required. :woman_facepalming: